Taskfile.yml

```diff
@@ -0,0 +1,87 @@
+version: "3"
+
+includes:
+  utils: "tools/yscope-dev-utils/taskfiles/utils.yml"
+
+vars:
+  # Paths
```

Comment:
Not strictly necessary right now since we only have one group, but harmless to leave it.